tests/forms_tests/tests/test_formsets.py

```diff
@@ -1123,6 +1046,10 @@ def clean(self):
                 raise ValidationError("This is a non-form error")
 
         ChoiceFormSet = formset_factory(Choice, formset=BaseCustomFormSet)
+        data = {
+            'choices-TOTAL_FORMS': '1',
+            'choices-INITIAL_FORMS': '0',
+        }
```

Comment:
Previously it used `data` defined in [outer scope](https://github.com/django/django/pull/11071/files#diff-d68339db96d32ee0bec372db9f20ded5L1183).